docs/earthfile/earthfile.md

```diff
@@ -181,6 +181,17 @@ release:
     RUN --push --secret GITHUB_TOKEN=+secrets/GH_TOKEN github-release upload
 ```
 
+##### `--ssh`
+
+Allows a command to access the ssh authentication client running on the host via the socket which is referenced by the environment variable `SSH_AUTH_SOCK`.
+
+Here is an example:
+
+```Dockerfile
+git-clone:
+    RUN --ssh git clone git@host.com:user/private-repo.git
```

Comment:
I just realized that we give an example of an operation that we generally recommend to be done via our own `GIT CLONE` command. Perhaps replace with `go mod download`?

```suggestion
    RUN --ssh go mod download
```

Reply:
that's a really good point. If we want to use go, then we might also have to change it to:
`RUN --ssh git config --global url."git@github.com:".insteadOf "https://github.com/" && go mod download`

Reply:
SGTM